doc/python/format.rst

```diff
@@ -0,0 +1,80 @@
+.. _pycablecellformat:
```

Comment:
Currently, (links to) formats live in `doc/python/morphology.rst`. Maybe that should link here instead, and the other formats listed here as well.

Same for the Concepts section.

Reply:
The formats that live in `doc/python/morphology.rst` are morphology specific (SWC and neuroML) so it makes sense for them to live there. Of course we could place them under this section, I'd rather not do this in this PR though, as it is already quite big.

Reply:
Sure, but I'm not sure too much nesting is helping figure out how supported file formats work. A separate `format.rst` shows up in the TOC, and thus will be the first place to look, and the first place to list the formats. It's actually not that much text that would move.

Reply:
I agree. I'll make sure to move them right after this PR is merged, in a separate PR, just to keep the scope of this one limited. Does that sound good?

Reply:
:+1: